compiler/damlc/tests/src/DA/Test/Packaging.hs

```diff
@@ -1987,6 +1757,37 @@ dataDependencyTests Tools{damlc,repl,validate,davlDar,oldProjDar} = testGroup "D
         withCurrentDirectory (tmpDir </> "proj") $
             callProcessSilent damlc ["build"]
     ]
+  where
+    simpleImportTest :: String -> [String] -> [String] -> TestTree
+    simpleImportTest title importee importer =
+        testCaseSteps title $ \step -> withTempDir $ \tmpDir -> do
+            step "building project to be imported via data-dependencies"
+            createDirectoryIfMissing True (tmpDir </> "type")
+            writeFileUTF8 (tmpDir </> "type" </> "daml.yaml") $ unlines
+                [ "sdk-version: " <> sdkVersion
+                , "name: type"
+                , "source: ."
+                , "version: 0.1.0"
+                , "dependencies: [daml-prim, daml-stdlib]"
+                ]
+            writeFileUTF8 (tmpDir </> "type" </> "P1.daml") $ unlines importee
```

Comment:
`P1` and `P2` seems unnecessarily confusing. Why not `Dep` and `Main` or `Importer` and `Importee`?

Reply:
I thought about that as well. I'll change it to `Lib` and `Main`, I'm worried `Importer` and `Importee` are too close to get mixed up.

Reply:
The `automerge` label was faster than me... Here we go: https://github.com/digital-asset/daml/pull/7665

Reply:
I'm not gaming my PR stats!

Reply:
looking at the contributor stats you should :trollface: